spec/cases_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'spec_helper'
 require 'json'
 
-describe MessagePack do
+RSpec.describe MessagePack do
```

Comment:
I'm not sure about it. It's introduced by `synvert` rule.
It looks to reduce `Kernel` modification dependency as much as possible. Not bad.